spec/configuration/prison_data_spec.rb

```diff
@@ -20,18 +20,48 @@
   context "enabled prisons" do
     let(:enabled) { Prison.enabled }
 
+    it "currently number between 95 and 105" do
+      expect(enabled.count).to be_within(5).of(100)
+    end
+
     it "each has a nomis_id" do
       expect(enabled.any?{ |p| p.nomis_id.blank? }).to be_falsey
     end
 
-    it "each unbookable date for each prison should be a valid date" do
-      unbookable = enabled.map(&:unbookable).flatten
-      expect(unbookable.map(&:class).all?{ |p| p == Date }).to be_truthy
-    end
+    context 'unbookable dates' do
+      it "are all valid dates" do
+        unbookable = enabled.flat_map(&:unbookable)
+        expect(unbookable.map(&:class).all?{ |p| p == Date }).to be_truthy
```

Comment:
We can use [the `all` matcher](https://relishapp.com/rspec/rspec-expectations/v/3-4/docs/built-in-matchers/all-matcher) here:

```ruby
expect(unbookable).to all(be_a(Date))
```